scipy/stats/qmc.py

```diff
@@ -23,6 +23,7 @@
    Sobol
    Halton
    LatinHypercube
+   OptimalLatinHypercube
```

Comment:
Need to remove `_qmc.pyi` above, I think?

Reply:
Totally! A left over from a merge I guess.

Reply:
Removed.